### DISCUSSION TITLE
Intel AX210 test results

### DISCUSSION THREAD

**Discussion by lunananananananana** [vanilla-wiiu/vanilla#22] [Hardware Results]
I'm using an Intel AX210, installed in a Framework 16 (note that this isn't the stock card, I swapped it from a Mediatek one)

I was easily able to pair and connect, and video, audio, and input all works. No lag in the video stream either, but I assume that's more of a GPU thing than wifi card

**Comment by VoxelTek** [comment]
Cool, I'll add this to the compatibility table! 👍🏻

**Comment by TCAtrevor** [comment]
This chipset isn't working for me right now, there's no audio or video being sent to Vanilla.

**Reply by VoxelTek** [reply]
To confirm, you are using the AX210? Could you specify more about your setup? (e.g. what operating system)

**Reply by TCAtrevor** [reply]
I was running it on EndeavourOS on a custom PC.

**Reply by TCAtrevor** [reply]
From Windows I can confirm it is an AX210.
![image](https://github.com/user-attachments/assets/9bc4c9ea-4cb2-4de5-83f5-f727c9fe3ad7)

**Reply by ryzobyte** [reply]
I can't get this chipset to work either. Vanilla says "Connecting to Wii U 1..." and the Wii U recognizes that a gamepad is connected, but Vanilla is stuck on that screen.

**Reply by itsmattkc** [reply]
> I can't get this chipset to work either. Vanilla says "Connecting to Wii U 1..." and the Wii U recognizes that a gamepad is connected, but Vanilla is stuck on that screen.

Are you able to send run Vanilla in a Terminal and send the resulting logs?

**Reply by ryzobyte** [reply]
[ax-210-1-1-26.log](https://github.com/user-attachments/files/24402300/ax-210-1-1-26.log)

**Reply by itsmattkc** [reply]
It looks like you're missing the VAAPI drivers for your system.

Assuming you're using an Intel GPU, you probably need to install `intel-media-va-driver` (Debian/Ubuntu/Mint) or `intel-media-driver` (Fedora).

**Reply by ryzobyte** [reply]
<img width="631" height="122" alt="image" src="https://github.com/user-attachments/assets/1fc2d824-d80d-4a15-b992-135e1a306b0d" />
Mint shows me that the driver is already installed. I'm using a live USB if that makes a difference.

**Reply by ryzobyte** [reply]
I'm also using an NVIDIA GPU rather than Intel.

**Reply by itsmattkc** [reply]
NVIDIA complicates things a little. If you're on a live USB, you're probably using the limited open source Nouveau driver rather than the official NVIDIA driver. But even with the official driver, there also appear to be some issues specific to NVIDIA cards at the moment (#271, #258) that I haven't had a chance to diagnose yet.

I should hopefully have those resolved at some point, I'll update this thread when I do.

**Reply by A5Tr0-Dr0n3** [reply]
GUYS!!! I GOT FROUNT END WORKING ON WINDOWS!!! At least I think. I have the same Wifi Adaptor in my PC, and the Wifi card is not appearing in the app, BUT THE APP IS OPENING!
<img width="1919" height="1079" alt="image" src="https://github.com/user-attachments/assets/c0d8a9c2-17cd-4a72-948e-4fd5087bf1d7" />

**Comment by EMREOYUN** [comment]
I also switched from MediaTek to Intel AX210 but I don't have the Wii U with me so I cannot do tests with AX210 until somehow I find a Wii U.
